cmd/agent.go

```diff
@@ -117,4 +117,11 @@ func init() {
 		os.Getenv("API_TOKEN"),
 		"Token used for authentication when API tokens are in use on the backend",
 	)
+	agentCmd.PersistentFlags().BoolVarP(
+		&agent.Profiling,
+		"profiling",
```

Comment:
```suggestion
		"enable-pprof",
```

I think this would be a better flag to describe what it does.